addons/point_of_sale/models/pos_order.py

```diff
@@ -253,6 +255,8 @@ def _get_pos_anglo_saxon_price_unit(self, product, partner_id, quantity):
     session_move_id = fields.Many2one('account.move', string='Session Journal Entry', related='session_id.move_id', readonly=True, copy=False)
     to_invoice = fields.Boolean('To invoice')
     is_invoiced = fields.Boolean('Is Invoiced', compute='_compute_is_invoiced')
+    is_tipped = fields.Boolean('Is this already tipped?')
```

Comment:
Do you think we need migrations? check in existing orders that if they have the product which is defined as tip product on the related pos config, then we can set that it is tipped, and set the amount of the line?

Reply:
I was thinking that it may not be necessary but now that you mentioned it, it might be important once reporting is modified to include tipped amounts. I'll write the migration script.